src/main/java/com/rackspacecloud/metrics/ingestionservice/listeners/processors/TenantIdAndMeasurement.java

```diff
@@ -0,0 +1,16 @@
+package com.rackspacecloud.metrics.ingestionservice.listeners.processors;
+
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@RequiredArgsConstructor
```

Comment:
Doesn't seem to be needed since you declared a constructor.

Reply:
Submitted fixes